.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__/
 *.pyc
 venv/
+
```

Comment:
is that really necessary? you might wanna remove that from the pull request...

Reply:
Done!